src/test-run/index.js

```diff
@@ -157,10 +158,17 @@ export default class TestRun extends Session {
         this.pendingDriverTask = null;
     }
 
+    _clearPendingRequest () {
+        if (this.pendingRequest)
+            clearTimeout(this.pendingRequest.responseTimeout);
+
+        this.pendingRequest = null;
```

Comment:
Well, I guess it makes sense to move this assignment into condition body as well